src/common/interfaces.ts

```diff
@@ -59,4 +42,4 @@ export interface IJobResponse<P, T> {
   expirationDate?: Date;
```

Comment:
is this interface needed? "IJobResponse" - can it be taken from mc-priority-queue?

Reply:
done. removed and imported now from mc-models